.github/workflows/ci.yml

```diff
@@ -41,7 +41,9 @@ jobs:
         with:
           node-version: 12
       - name: Install packages
-        run: npm ci && npm ci --prefix test
+        run: |
```

Comment:
Out of curiosity: Why are you using the pipe operator? I'm not that into yaml but I've always thought `>` is the default for multi-line strings.

Reply:
This is the multi-line operator actually. [stackoverflow](https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml)